package.json

```diff
@@ -16,12 +16,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.19.0",
     "babel-core": "^6.17.0",
+    "babel-preset-env": "^1.7.0",
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
     "faker": "^4.1.0",
     "jquery": "^3.1.1",
```

Comment:
If you're using axios as your choice of making HTTP requests then you can npm uninstall jquery from your dependencies, just to make your package.json look cleaner.